### ISSUE TITLE
Skip logic within group

### ISSUE THREAD

**Issue by yanokwa** [clint-tseng/odkbuild#33]
From brijsde@gmail.com:

"I am trying to develop a data collection form with ODK build with some questions in a group. If I add skip logic to questions outside the group or if i add skip logic to the whole group, it works fine but if I add skip logic to questions within the group, it does not work. If I take same question out of the group, the same logic starts working."

**Comment by clint-tseng** [comment]
@yanokwa any idea what this is specifically referring to?

**Comment by yanokwa** [comment]
@clint-tseng This is the link to the original post https://groups.google.com/d/msg/opendatakit-developers/2C1VC6JNKAI/X-xcF283-7IJ.

Mitch says (and I agree) that "the questions that should be shown on a page are determined prior to rendering the page, and this set of visible prompts is not updated until you navigate off the page and back onto it. So if there is skip logic that says that question B should not be shown if question A has certain values, that logic will not be executed while you are within the screen that displays the field-list group containing questions A and B."

TLDR: Skip logic will not be reliable if you use the field-list appearance to display dependent the questions on a single screen.

**Comment by clint-tseng** [comment]
ah ha, it's field-list that's the problem. I can validate out skip logic when nested in a field-list.
